src/js/menu/menu-item.js

```diff
@@ -17,7 +17,18 @@ class MenuItem extends Button {
 
   constructor(player, options) {
     super(player, options);
+
+    this.selectable = options['selectable'];
+
     this.selected(options['selected']);
+
+    if (this.selectable) {
+      // TODO: May need to be either menuitemcheckbox or menuitemradio,
+      //       and may need logical grouping of menu items.
+      this.el_.setAttribute('role', 'menuitemcheckbox');
```

Comment:
Not sure if this is a chromevox issue or not, but if I have captions in the list, when I expand the captions menu, it'll read the label of the last caption in the list.